README.md

```diff
@@ -76,7 +76,9 @@ The OpenGApps AOSP based build system is placed in `vendor/google/build` and inc
 in `build/core/main.mk`:
 ```
 # Include the google-specific config
+ifneq ($(GAPPS_VARIANT),)
```

Comment:
If any device.mk hava set GAPPS_VARIANT, but another device.mk not set GAPPS_VARIANT,In Cyanogenmod,If I common lunch cm_device-userdbug && make -j9,shell will print "error GAPPS_VARIANT was not found. Use of one of pico,nano,micro,mini,full,stock,super)"